src/main/java/hudson/remoting/ObjectInputStreamEx.java

```diff
@@ -41,6 +41,7 @@ public class ObjectInputStreamEx extends ObjectInputStream {
     /**
      * @deprecated as of 2.53
      */
+    @Deprecated
```

Comment:
Just wanted to be able to `mvn test` without getting distracted by compiler warnings.